### PR TITLE
Updates to governments.yml

### PR DESCRIPTION
**Pull request by jcastle-zz**
Manually checked each in the list and validated for consistency as federal government org.  My findings are reflected in the new yml file with rationale provided below.

- adl-aicc - defense, moved to Military list
- adlnet - defense, moved to Military list
- afrl - air force research lab, moved to Military list
- arcticlcc - nonprofit, moved to civic_hackers.yml
- IIP-Design - Infosys design lab, deleted from yml file
- llnl - part of DOE, added to governments.yml
- missioncommand - US Army Mission Command, moved to military list
- NationalGuard - military, moved to Military list
- opengovplatform - nonprofit, moved to civic_hackers.yml
- ozoneplatform - Ozone private company, deleted from list
- redhawksdr - GEON Technologies llc - private, deleted from list
- SERVIR - SERVIR Global - private, deleted from list
- virtual-world-framework - defense, moved to U.S. Military section
- info-sharing-environment - DHS, not considered Intelligence agency, move to general gov't list
- project-interoperability - DHS, not considered Intelligence agency, move to general gov't list
- shareandprotect - DHS, not considered Intelligence agency, move to general gov't list

### CODE REVIEW

File: _data/governments.yml [hunk 1]
```diff
@@ -639,20 +634,17 @@ U.S. Federal:
...
   - ombegov
   - Open-Sat
-  - opengovplatform
-  - ozoneplatform
   - peacecorps
   - petsc
   - presidential-innovation-fellows
+  - project-interoperability
   - project-open-data
   - radiofreeasia
```
Comment (jcastle-zz) [inline _data/governments.yml:651]: radiofreeasia needs to be moved to the civic_hackers.yml list.  I missed it here.

**Comment by nschonni** [comment]
A chunk of this is synced using this script https://github.com/github/government.github.com/blob/gh-pages/script/fetch-us so it may get reverted. Not sure how to fix the upstream site though.

File: _data/governments.yml [hunk 2]
```diff
@@ -604,26 +600,25 @@ U.S. Federal:
   - HHSDigitalMediaAPIPlatform
   - HHSIDEAlab
   - historyatstate
```
Comment (jcastle-zz) [inline _data/governments.yml:611]: Also need to add HUD - https://github.com/hudgov

**Comment by jbjonesjr** [comment]
> Manually checked each

👀  👀  👀  👀  👀  👀

File: _data/governments.yml [hunk 3]
```diff
@@ -639,20 +634,17 @@ U.S. Federal:
   - ntia
   - ombegov
   - Open-Sat
-  - opengovplatform
-  - ozoneplatform
```
Comment (jbjonesjr) [inline _data/governments.yml:652]: you sure on ozone being a private company?

Do you have new intel on that?

**Review by jbjonesjr — CHANGES_REQUESTED** [review]

**Comment by jbjonesjr** [comment]
fyi, we'll get this PR straightened out (and hopefully look into cleaning some of the upstream digitalgov data), and then we can look at the related civic data checks.

re: manual checking
- i've been thinking recently about adding a CI check to ensure that the repos have at least 1 public repo. That would be a helpful check to add.
- what other checks did you do @jcastle ? Anything we can automate (even if it's not part of the CI job, but at least a check that could be run out of band?)

**Comment by jcastle-zz** [comment]
- At least one public repo would be good for gov't OSS
- Link in GH org, http://ozoneplatform.org, goes to https://ozone.nextcentury.com/.  This is a commercial website selling ozone services.
- All checks were manual - looked at org hyperlink (agency.gov), poked around org website if needed, then Google search; unfortunately no automation to determine list of gov't orgs

File: _data/governments.yml (hunk 3 above)
Comment (jcastle-zz) [reply _data/governments.yml:652]: See comment below on ozoneplatform

Comment (jbjonesjr) [reply _data/governments.yml:652]: Eh, it's a bit more complicated than that. it started as a Govt off-the shelf project, was put into the public sphere as part of the [FY2012 NDAA (sec. 924)](https://www.gpo.gov/fdsys/pkg/PLAW-112publ81/pdf/PLAW-112publ81.pdf), was then managed by a Government Open Source Managed board (GOSS as it was called at the time), but has recently been transitioned to Next Century as custodian of the efforts. In reality, I think the framework is mostly deprecated and no longer used by many in the DoD/IC community, but I think through the 
 Cooperative Research and Development Agreements (CRADA) that denotes Next Century (ozoneplatform.org) as the custodian of Ozone, I think it's still :+1:.

That said, i'll let you make the final decision.

Comment (jcastle-zz) [reply _data/governments.yml:652]: Since it goes to a .com and they are selling commercial services, I think we should take them out of the US gov't list.

**Comment by jcastle-zz** [comment]
One more @jbjonesjr for the National Science Foundation - nsf-open

File: _data/governments.yml [hunk 4]
```diff
@@ -610,26 +606,25 @@ U.S. Federal:
...
   - nasa
   - nasa-develop
   - nasa-gibs
   - NASA-rdt
   - nasa-tournament-lab
   - nasaworldwind
-  - NationalGuard
   - nationalparkservice
   - NCRN
   - NeoGeographyToolkit
```
Comment (jcastle-zz) [inline _data/governments.yml:633]: Need to include National Science Foundation - nsf-open

**Comment by jbjonesjr** [comment]
what's your thought here on getting this merged @jcastle ? Want me to make these last changes or do you have it?

File: _data/governments.yml (hunk 3 above)
Comment (jbjonesjr) [reply _data/governments.yml:652]: fair enough

**Review by jbjonesjr — APPROVED** [review]

**Comment by froi** [comment]
Hey @jcastle I'd like to get this squared away. Any chance we can circle back on this?

**Comment by jcastle-zz** [comment]
@froi, this is pretty old now and probably not the latest. I would go with what is currently on the site.

**Comment by froi** [comment]
No worries @jcastle, thanks for the comment.

Closing due to inactivity and time.

**Pull request closed without merging**